### PR TITLE
Feature/figma tokens ihor UuiRangeDatePicker

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: epam-promo/components/datePickers/index.ts [hunk 1]
```diff
@@ -1,6 +1,2 @@
-export * from './Calendar';
-export * from './CalendarPresets';
 export * from './DatePicker';
-export * from './DatePickerBody';
 export * from './RangeDatePicker';
-export * from './RangeDatePickerBody';
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/datePickers/index.ts:6]: They imports from uui

File: loveship/components/datePickers/index.ts [hunk 2]
```diff
@@ -1,6 +1,2 @@
-export * from './Calendar';
-export * from './CalendarPresets';
 export * from './RangeDatePicker';
 export * from './DatePicker';
-export * from './RangeDatePickerBody';
-export * from './DatePickerBody';
```
[the remark this answers is not in the document]
Comment (vik753) [reply loveship/components/datePickers/index.ts:6]: They imports from uui

File: epam-promo/components/datePickers/docs/RangeDatePicker.doc.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@use  '../../../assets/styles/index' as *;
+
+.container {
+    display: flex;
+    padding: 9px 30px;
+    justify-content: space-between;
+    border-top: 1px solid var(--uui-rdtp-border-color);
+    align-items: center;
+    background-color: var(--uui-rdtp-bg-color);
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/datePickers/docs/RangeDatePicker.doc.scss]: done

File: uui/components/datePickers/RangeDatePicker.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                    this.props.isInvalid && uuiMod.invalid,
+                    this.state.inFocus && uuiMod.focus,
+                ) }
+                onClick={ !this.props.isDisabled && props.onClick }
+                onBlur={ this.handleWrapperBlur }
+                ref={ props.ref }
+            >
+                <TextInput
+                    icon={ systemIcons[this.props.size || '36'].calendar }
+                    cx={ cx(this.props.bodyCx, 'date-range-picker-vars', css.dateInput, css['size-' + (this.props.size || 36)], this.state.inFocus === 'from' && uuiMod.focus) }
```
[the remark this answers is not in the document]
Comment (vik753) [reply uui/components/datePickers/RangeDatePicker.tsx]: done

File: epam-promo/components/filters/FilterRangeDatePickerBody.tsx [hunk 5]
```diff
@@ -2,10 +2,11 @@ import * as React from 'react';
 import dayjs from "dayjs";
 import { BaseRangeDatePickerProps, IDropdownBodyProps, RangeDatePickerInputType, uuiMod } from "@epam/uui-core";
 import { BaseRangeDatePicker } from '@epam/uui-components';
-import { FlexRow, i18n, RangeDatePickerBody, TextInput, LinkButton, FlexSpacer, FlexCell } from '../../index';
+import { FlexRow, i18n, TextInput, LinkButton, FlexSpacer, FlexCell } from '../../index';
 import cx from "classnames";
-import css from "../datePickers/RangeDatePicker.scss";
+import css from "@epam/uui/components/datePickers/RangeDatePicker.scss";
```
Comment (AlekseyManetov) [inline epam-promo/components/filters/FilterRangeDatePickerBody.tsx:7]: create separate styles file inside promo

File: uui/components/datePickers/DatePicker.tsx [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
+import { DatePickerBody } from "./DatePickerBody";
 import { systemIcons } from "../../icons/icons";
-import css from "./DatePicker.scss";
-import './DatePicker.colorvars.scss';
+import { DropdownContainer } from "../overlays";
 
 const defaultMode = EditMode.FORM;
 
-export interface DatePickerProps extends DatePickerCoreProps, SizeMod, IHasEditMode {}
+export interface UuiDatePickerProps extends DatePickerCoreProps, SizeMod, IHasEditMode {}
```
Comment (AlekseyManetov) [inline uui/components/datePickers/DatePicker.tsx:12]: rename to DatePickerProps

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

**Pull request merged**